.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-    - 1.x
+    - master
```

Comment:
> ideally just need Go 1.9+

You'll have to try this to confirm, but I think there's a chance it'll work. Try using a tag name, i.e., `1.9rc1`.

Reference: https://github.com/travis-ci/gimme/blob/df601a365f8313f3a09167892f9425946927a728/gimme#L102.